<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerSettings</class>
 <widget class="QWidget" name="ServerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>436</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Collection:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="combo"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="addButton"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="removeButton"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="1" column="0">
          <widget class="BuddyLabel" name="hostLabel">
           <property name="text">
            <string>Host (or local socket):</string>
           </property>
           <property name="buddy">
            <cstring>host</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="LineEdit" name="host"/>
         </item>
         <item row="2" column="0">
          <widget class="BuddyLabel" name="portLabel">
           <property name="text">
            <string>Port:</string>
           </property>
           <property name="buddy">
            <cstring>port</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="SpinBox" name="port">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>6600</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="BuddyLabel" name="passwordLabel">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="buddy">
            <cstring>password</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="LineEdit" name="password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="BuddyLabel" name="dirLabel">
           <property name="text">
            <string>Music folder:</string>
           </property>
           <property name="buddy">
            <cstring>dir</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="PathRequester" name="dir"/>
         </item>
         <item row="6" column="0">
          <widget class="BuddyLabel" name="dynamizerPortLabel">
           <property name="text">
            <string>Dynamizer port:</string>
           </property>
           <property name="buddy">
            <cstring>dynamizerPort</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="SpinBox" name="dynamizerPort">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>6600</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="BuddyLabel" name="coverNameLabel">
           <property name="text">
            <string>Cover filename:</string>
           </property>
           <property name="buddy">
            <cstring>coverName</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="LineEdit" name="coverName"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="buddy">
            <cstring>name</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="LineEdit" name="name"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="musicFolderNoteLabel">
         <property name="text">
          <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; The 'Music folder' setting is used to lookup cover-art (either the value specified above, or &lt;code&gt;cover.jpg / cover.png&lt;/code&gt; if not set). If you also have write permissions to this folder (and it's sub-folders), then Cantata will save any downloaded covers into the respective album folder.&lt;/i&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="socketNoteLabel">
         <property name="text">
          <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; When using a local socket the full absolute path to the socket needs to be set. (Port number is not required.)&lt;/i&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="dynamizerNoteLabel">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="4">
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="BuddyLabel" name="dirLabel_2">
           <property name="text">
            <string>Music folder:</string>
           </property>
           <property name="buddy">
            <cstring>dir</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="PathRequester" name="basicDir"/>
         </item>
         <item row="1" column="0">
          <widget class="BuddyLabel" name="basicCoverNameLabel">
           <property name="text">
            <string>Cover filename:</string>
           </property>
           <property name="buddy">
            <cstring>basicCoverName</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="LineEdit" name="basicCoverName"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="StateLabel" name="basicMusicFolderNoteLabel">
         <property name="text">
          <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; If you change the 'Music folder' setting, then you will need to manually update the music database. This can be performed by pressing the 'Refresh Database' button in the 'Artists' or 'Albums' views.&lt;/i&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QLabel" name="musicFolderNoteLabel_2">
         <property name="text">
          <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; If no setting is specified for 'Cover filename, then Cantata will use a default of &lt;code&gt;cover&lt;/code&gt; &lt;/i&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>363</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>buddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>StateLabel</class>
   <extends>QLabel</extends>
   <header>statelabel.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>PathRequester</class>
   <extends>QLineEdit</extends>
   <header>pathrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
